scylla-cdc/src/stream_generations.rs

```diff
@@ -238,15 +234,14 @@ async fn get_cluster_size(session: &Session) -> anyhow::Result<usize> {
     // We are using default consistency here since the system keyspace is special and
     // the coordinator which handles the query will only read local data
     // and will not contact other nodes, so the query will work with any cluster size larger than 0.
-    let mut rows = session
-        .query("SELECT COUNT(*) FROM system.peers", &[])
+    let (rows,) = session
+        .query_unpaged("SELECT COUNT(*) FROM system.peers", &[])
         .await?
-        .rows
-        .unwrap()
-        .into_typed::<(i64,)>();
+        .into_rows_result()?
+        .first_row::<(i64,)>()?;
 
     // Query returns a number of peers in a cluster, so we need to add 1 to count current node.
-    Ok(rows.next().unwrap().unwrap().0 as usize + 1)
+    Ok(rows as usize + 1)
```

Comment:
:wrench: the variable name should be changed, because it's no more `rows` that is represented by the variable.
I suggest `peers_num`.